__docs__/wonder-blocks-dropdown/combobox.stories.tsx

```diff
@@ -323,6 +327,24 @@ export const ControlledMultilpleCombobox: Story = {
             <View style={styles.wrapper}>{Story()}</View>
         ),
     ],
+
```

Comment:
**note:** Moved `play` from the previous story to the new one as it made sense to snapshot the controlled version of the multi-select mode. That way, the default multi-select version appears closed on load: 
https://5e1bf4b385e3fb0020b7073c-uwiczkggys.chromatic.com/?path=/story/packages-dropdown-combobox--multiple-selection

| BEFORE | AFTER |
|--------|--------|
| <img width="929" alt="Screenshot 2024-08-29 at 5 43 05 PM" src="https://github.com/user-attachments/assets/0046a7ea-f2df-43f0-97fd-aa9699461ae2"> | <img width="1005" alt="Screenshot 2024-08-29 at 5 43 12 PM" src="https://github.com/user-attachments/assets/ef2448af-806b-4a94-a8b2-c7f11143b981"> |

Before this change, both stories, the default one and the controlled one were throwing a very similar snapshot. Now both snapshots capture different states.